src/data/equiv/fin.lean

```diff
@@ -56,12 +56,12 @@ def sum_fin_sum_equiv : fin m ⊕ fin n ≃ fin (m + n) :=
       from (nat.add_sub_of_le $ le_of_not_gt H).symm ▸ x.2⟩,
   left_inv := λ x, begin
     cases x with y y,
-    { simp [y.2, fin.ext_iff] },
+    { simp [y.2, fin.ext_iff, y.is_lt], },
```

Comment:
Aren't `y.2` and `y.is_lt` the same thing?

Reply:
`y.is_lt` uses the coe.

Reply:
Do you still need `y.2` though? I think I will need to edit this particular function soon anyways, so I guess I don't really care.

Reply:
Fixed, good catch.